src/app/helptext/vm/vm-wizard/vm-wizard.ts

```diff
@@ -78,6 +78,7 @@ volsize_validation: [Validators.required, Validators.min(1)],
 pool_detach_warning_paraText: T("Select zvol"),
 
 datastore_tooltip: T('Choose a pool or dataset for the new zvol.'),
+datastore_placeholder: T('Select zvol'),
```

Comment:
As stated in the ticket 102200, this label is wrong, we are not selecting a zvol, we are selecting a dataset to put a zvol on.  William's suggestion was "Filesystem to create a volume".  Please work with docs on getting a better label for this.

Reply:
@wblock @Mrt134 could one of you come up with a good label for this?